Dockerfile

```diff
@@ -18,7 +18,8 @@ VOLUME ${APACHEDS_DATA}
 
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections \
     && apt-get update \
-    && apt-get install -y ldap-utils procps openjdk-7-jre-headless curl jq \
+    && apt-get install -y apt-utils \
+    && apt-get install -y ldap-utils procps openjdk-8-jre-headless curl jq \
```

Comment:
Would you mind rewriting this so it'll be easier to make future changes:
```
&& apt-get install -y \
    apt-utils \
    ldap-utils \
    ...
```

Reply:
Fixed